### ISSUE TITLE
Decouple the search indexing on mobile

### ISSUE THREAD

**Issue by pixernot** [KeypDemoCorp/awesome-ui#92]
The current behavior is inconsistent across browsers and needs to be addressed.

Accessibility should be considered when implementing this.
